saleor/discount/migrations/0051_detach_sale_from_permission.py

```diff
@@ -92,4 +92,17 @@ class Migration(migrations.Migration):
             """,
             reverse_sql=migrations.RunSQL.noop,
         ),
+        migrations.SeparateDatabaseAndState(
+            state_operations=[
+                migrations.AlterModelOptions(
+                    name="promotion",
+                    options={
+                        "ordering": ("name", "pk"),
+                        "permissions": (
+                            ("manage_discounts", "Manage sales and vouchers."),
```

Comment:
Just wondering if we should keep `sales` name here instead of `Manage promotions and vouchers.` ?

Reply:
This name is stored in the database. It could be fixed in a separate PR.